pwa-devdocs/src/product-recs/index.md

```diff
@@ -0,0 +1,88 @@
+
+## Install the Product Recommendations module
+
+Product Recommendations support on PWA requires installing the `venia-product-recommendations` package and the Product Recommendations Magento module.
+
+1. You can install the PWA `venia-product-recommendations` package from the NPM registry:
+
+   ```sh
+   npm install @magento/venia-product-recommendations
+   ```
+   This package contains everything you need to collect behavioral data and render the recommendations. Some recommendation types use behavioral data from your shoppers to train machine learning models that build personalized recommendations. Other recommendation types use catalog data only and do not use any behavioral data. See the [Magento user guide](https://docs.magento.com/user-guide/marketing/product-recommendations.html#trainmlmodels) to learn how Adobe Sensei trains machine learning models that results in higher quality recommendations.
+
+1. To install the Magento Product Recommendations backend module, see the [Magento developer documentation](https://devdocs.magento.com/recommendations/install-configure.html).
+
+1. Additionally, you need to install the `module-data-services-graphql` module that expands Magento's existing GraphQL API to include fields that are used to fetch the recommendations from the Recommendations Service.
+
+   ```sh
+   npm install @magento/module-data-services-graphql
+   ```
+
+## Create recommendation units
+
+Creating a product recommendation unit for your PWA storefront is similar to [creating one for a Magento storefront](https://docs.magento.com/user-guide/marketing/create-new-rec.html).
+The difference is that after you create a recommendation unit, you then need to add code to your PWA storefront to explicitly render the recommendation unit from the Magento backend.
+
+## Render recommendations
+
+To control the recommendation unit placement on your PWA storefront, use the local intercept file. See the [extensibility framework][] documentation to learn more. To render the recommendation unit on your PWA storefront, use one of the methods described below.
+
+### Venia UI component
+
+The `Recommendations` component is part of the [venia-ui package][].
+It contains React components that do the following:
+
+-  Collect and send behavioral data to Adobe Sensei
+-  Fetch recommendations from the recommendations service
+-  Render the recommendation unit to your storefront page
+
+#### Example
+
+```jsx
+//Render recommendations using visual component
+return <Recommendations pageType={CMS} />
+```
+### Fetch data only
+
+If you are not using the `venia-ui` package, you can call the recommendations service and receive a JSON payload.
+#### Example
+
+```js
+import { Pagetypes, useRecsData, VeniaProductRecommendations } from "@magento/venia-product-recommendations"
+const recs = useRecsData({pagetype: Pagetypes.CMS}) 
+// or
+return <VeniaProductRecommendations pageType={Pagetypes.PRODUCT} />
+```
```

Comment:
`// or` indicates these are two examples, they should be split into two different code blocks with explanations on what each does

Reply:
<VeniaProductRecommendations> actually renders the recommendations and as such doesn't belong to this section. The only method to fetch data only is to use the `useRecsData` hook. So change the code snippet to: 

```js
import { Pagetypes, useRecsData, VeniaProductRecommendations } from "@magento/venia-product-recommendations"
const {data, error, isLoading} = useRecsData({pagetype: Pagetypes.CMS}) 
```